zuul.d/jobs.yaml

```diff
@@ -1174,3 +1174,19 @@
       - moo_github_account
     vars:
       is_gpu_enabled: true
+
+- job:
+    name: containerd-build-arm64-periodic
```

Comment:
it is better do not add the `periodic` field in jobs' name, there is a doc described about how to name jobs. 
https://docs.openlabtesting.org/publications/contributors/development-guide#job-naming-notations

Reply:
+1

Reply:
done